usr/src/lib/libzpool/Makefile.com

```diff
@@ -61,6 +61,7 @@ CLEANFILES += $(EXTPICS)
 
 $(LINTLIB) := SRCS=	$(SRCDIR)/$(LINTSRC)
 $(LINTLIB): ../common/zfs.h
```

Comment:
I'm not using targets $(LINTLIB) with dilos builds - i have no dependencies to SunStudio and no need it.